src/components/Input.js

```diff
@@ -200,6 +200,11 @@ class Input extends Component {
 
   _value = () => this.state.value;
 
+  _updateValue = value => {
+    this.props.onChange(value);
```

Comment:
Yes because the component is the one that by default has an onChange, its not based on the user of package. 

By that what i mean it is optional for lets say `loot-web-app` to supply an onChange, but by default the components that invoke this function, will always have passed down an onChange to be executed.

This is an example of a component using the function, if you notice the onChange is passed down to the Input.

https://github.com/LootApp/loot-web-kit/blob/small-fix/src/components/InputDateTime.js#L127

The function is only for components where the user doesn't directly interact with input field, thus never triggering an `onChange`.